input/manure/manure_management.json

```
@@ -55,6 +55,13 @@
       "manure_handler": "flush system",
       "manure_separator": "rotary screen",
       "manure_treatment": "anaerobic digestion and lagoon with split"
+    },
+    {
+      "scenario_id": 8,
+      "bedding_type": "CBPB sawdust",
```

```suggestion
      "bedding_type": "compost bedded pack barn sawdust",
```
would it be better to be explicit with the name?

---

I did go back and forth between these two choices, but I went with the shorter version. Though, I'm open to changing it later.